guides/common/modules/con_about-running-jobs-on-hosts.adoc

```diff
@@ -8,7 +8,7 @@ For custom Ansible roles that you create, or roles that you download, you must i
 Before you can use Ansible roles, you must import the roles into {Project} from the {SmartProxy} where they are installed.
 
 Communication occurs through {SmartProxyServer}, which means that {ProjectServer} does not require direct access to the target host, and can scale to manage many hosts.
-Remote execution uses the SSH service that must be enabled and running on the target host.
+Remote execution uses the Script service that must be enabled and running on the target host.
```

Comment:
No matter how we call it in Foreman/Satellite, on it still does not change a name of the service running on the target host. On the target host it is still a ssh service (usually called sshd and being an ssh server from openssh). All in all this one should probably stay the way it was

Reply:
Hi, @adamruzicka !  Thanks for your review.  The ticket attached to this PR says "The ssh proxy feature will be renamed to script. All mentions of ssh in the installer will be renamed to script."

If these changes are all incorrect, what exactly is it that's being changed?  I'm going to have another PR for the same ticket/different guide, and I want to make sure that one is done correctly before submitting.  Thanks!

Reply:
> All mentions of ssh in the installer will be renamed to script

The installer had bunch of options looking like  `--foreman-proxy-plugin-remote-execution-ssh-$something`, those will now be `--foreman-proxy-plugin-remote-execution-script-$something`.

> The ssh proxy feature will be renamed to script.

This, as seen in Infrastructure > Capsules.

> what exactly is it that's being changed?

In addition to the two points above:
- provider type (as seen in Hosts > Job templates > $template > Job tab > Provider type)
- Names of default job templates will now have script instead of ssh in their names

And that should be it as far as I recall.